packages/matchbox/src/components/ListBox/ListBox.tsx

```diff
@@ -219,22 +261,23 @@ const ListBox = React.forwardRef(function ListBox(props, userRef) {
               onFocus={onFocus}
               onKeyDown={activatorKeyDown}
               disabled={disabled}
-              hasError={!!error}
+              $hasError={!!error}
               {...describedBy}
               ref={assignRefs}
             >
               {contentFromValue}
             </ListBoxButton>
-            <StyledChevron size={24} disabled={disabled} />
+            <StyledChevron size={24} $disabled={disabled} />
           </Box>
         }
       >
         <StyledList
           as="ul"
-          aria-activedescendant={currentValue}
+          aria-activedescendant={currentValue?.toString()}
```

Comment:
I don't think we have the optional-chaining plugin enabled do we? How is this working 🤔

Reply:
unless it gets included with `@babel/env`?

Reply:
I'm pretty sure it's enabled since babel 7.8.0